packages/std/src/math/uint256.rs

```diff
@@ -265,9 +265,9 @@ impl ops::Shr<u32> for Uint256 {
     type Output = Self;
 
     fn shr(self, rhs: u32) -> Self::Output {
-        if rhs > 256 {
+        if rhs >= 256 {
             panic!(
-                "right shift error: {} is larger than the number of bits in Uint256",
+                "right shift error: {} is larger or equal than the number of bits in Uint256",
```

Comment:
👍🏼 This is being handled in the other impls by delegating to the checked variant and unwrapping. Nice to have consistent behaviour here.

What about a checked_shr, by the way? I guess it's not currently needed / used. But wouldn't be bad to have; particularly because it doesn't panic.